modules/kotest/src/test/kotlin/org/testcontainers/containers/TestcontainerKotestIntegrationPerSpec.kt

```diff
@@ -0,0 +1,35 @@
+package org.testcontainers.containers
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class TestcontainerKotestIntegrationPerSpec : StringSpec({
+    val container = GenericContainer<Nothing>("ubuntu")
```

Comment:
let's not use `ubuntu`. If you don't care about the image, you can use the no-args constructor (it will be using `alpine`)

Reply:
Sure